modules/virt-importing-vm-wizard.adoc

```diff
@@ -72,13 +72,26 @@ ifdef::virt-importing-rhv-vm[]
 . Optional: You can select *Start virtual machine on creation*.
 endif::[]
 
-. Optional: Click *Edit* to update the following settings:
+. Click *Edit* to update the following settings:
 
 ifdef::virt-importing-rhv-vm[]
 * *General* -> *Name*: The VM name is limited to 63 characters. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857165[(*BZ#1857165*)]
 * *General* -> *Description*: Optional description of the VM.
-* *Storage* -> *Storage Class*: The default storage class must be NFS.
+ifeval::["{VirtVersion}" == "2.4"]
+* *Storage* -> *Storage Class*: The storage class must be *NFS*.
```

Comment:
Not sure how to write this the best way:
We can either pick NFS or set it o be default storage class, and not pick a storage class - that would pick NFS automatically.

and same refers to the bellow 2.5 storage settings.

Reply:
Will fix.